web/src/copy_and_paste.ts

```diff
@@ -666,18 +666,21 @@ export function try_stream_topic_syntax_text(text: string): string | null {
         return null;
     }
 
+    // Now we're sure that the URL is a valid stream topic URL.
+    // But the produced #**stream>topic** syntax could be broken.
+
     const stream = stream_data.get_sub_by_id(stream_topic.stream_id);
     assert(stream !== undefined);
     const stream_name = stream.name;
     if (topic_link_util.will_produce_broken_stream_topic_link(stream_name)) {
-        return null;
+        return topic_link_util.get_fallback_markdown_link(stream_name, stream_topic.topic_name);
```

Comment:
This case didn't have test coverage; I added a case that should hit it to the test suite.